droidlet/tools/crowdsourcing/vision_annotation_task/pilot_stats.py

```diff
@@ -280,3 +280,6 @@ def plot_scatter(
     if ymax:
         plt.ylim(0, ymax)
     plt.show()
+
+
+# %%
```

Comment:
what is the purpose of this line?

Reply:
It's superfluous, but it's added automatically when you run the cell above it.